aws.yaml

```diff
@@ -15,7 +15,7 @@
       version: 1.1.4
     - app: coredns
       componentVersion: 1.6.5
-      version: 1.1.6
+      version: 1.1.7
```

Comment:
Wait, this needs to be changed to master, not to aws-901-release branch

Reply:
For aws-901-release branch, 9.0.1 release definition has to be modified https://github.com/giantswarm/releases/pull/177/files#diff-c09dc41fbe3dfabd5f79fd91555907a9L356

Reply:
This is already changed in master - I think this will sort itself out when we merge PRs further. Nothing big to worry about IMO.